data/json/recipes/tools/containers.json

```diff
@@ -1195,6 +1195,31 @@
     "proficiencies": [ { "proficiency": "prof_leatherworking_basic", "time_multiplier": 2 } ],
     "components": [ [ [ "tanned_hide", 9 ] ], [ [ "filament", 80, "LIST" ] ] ]
   },
+  {
+    "result": "dry_bag_large",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "time": "6 h 40 m",
+    "autolearn": true,
```

Comment:
Is it reasonable to autolearn this at level three tailoring? And for a book to teach this at level 1?  I would think it would be more reasonable for the book to teach at 3-4 and autolearn to happen around 6-7?

Reply:
Yup, that's an overlook from my side.

Reply:
Added a fabrication requirement, too.